guides/common/modules/proc_importing-a-virtual-machine-from-proxmox-into-project.adoc

```diff
@@ -0,0 +1,12 @@
+[id="Importing_a_Virtual_Machine_from_Proxmox_into_{project-context}_{context}"]
+= Importing a Virtual Machine from Proxmox into {Project}
+
+You can import existing virtual machines running on Proxmox into {Project}.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Infrastructure > Compute Resources*.
+. Select your Proxmox compute resource.
+. On the *Virtual Machines* tab, click *Import as managed Host* or *Import as unmanaged Host* from the *Actions* menu.
```

Comment:
Should it be explained what should be clicked under what circumstances?

Reply:
Ideally yes, but it's copied from VMware which does not explain it either. I think that's something for a future improvement. :)